_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/create-sink-connector.adoc

```diff
@@ -0,0 +1,7 @@
+Now create a JDBC sink connector with the code below. Notice no transforms will be necessary to the `eventTime` column because the column is already a supported `TIMESTAMP` type.
+
++++++
+<pre class="snippet"><code class="sql">{% include_raw tutorials/connect-sink-timestamp/ksql/code/tutorial-steps/dev/create-sink-connector.sql %}</code></pre>
++++++
+
+Prior to ksqlDB 0.16.0, the only way to convert a `BIGINT` or `VARCHAR` to a `TIMESTAMP` was with https://docs.confluent.io/platform/current/connect/transforms/overview.html[Single Message Transforms (SMTs)].
```

Comment:
The information in the sentence is awesome, but the sentence feels misplaced/orphaned.  Any way to tie it in a bit more?  Either move it to another section, or keep it here and add a sentence (like...not verbatim...if you're going the SMT route, you'd add the SMT in this connector config above)

Reply:
hmm, thinking about where this should live.... I think near the top as part would make the most sense. Will comment once moved

Reply:
![Screen Shot 2021-03-17 at 8 44 02 AM (2)](https://user-images.githubusercontent.com/12937975/111486255-f37f0300-86fc-11eb-84a2-afe63d1d5223.png)

@ybyzek what are your thoughts on above?

Reply:
@awalther28 the callout is super cool 🥇   There is another drawback to SMTs -- they are not supported in Confluent Cloud -- but the KT should not draw attention to that ;) I do have minor edit suggestions, will drop it in the PR momentarily